### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by sanjayk0508**
## This PR contains
Updated the `README.md` file as requested. It now provides comprehensive information and includes helpful links for new users and contributors.

I have included the necessary points and information proposed by @toddysm. Please let me know if any additional information needs to be added to the `README.md` file.

#288

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,62 @@
-# Notary Project Website
+<br />
+<p align="center">
+  <img src="https://github.com/sanjayk0508/notaryproject.dev/assets/102804548/63986f73-9669-4efe-a571-4d1935e0a3d4" alt="image 36">
+  <br />
+    <br />
+    <b>Welcome to the Notary Project! Our website serves as a comprehensive resource for information and documentation on Notary. Powered by Hugo and hosted on Netlify, it's your go-to hub for all things related to Notary.</b>
```
Comment (iamsamirzon) [inline README.md:6]:
```suggestion
    <b>Welcome to the Notary Project! Our website, powered by Hugo and hosted on Netlify, serves as your go-to hub for all things related to Notary Project and its associated repositories.</b>
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,62 @@
...
 
-The [notaryproject.dev][] website, which contains information about the Notary Project as well as [documentation](https://notaryproject.dev/docs/) for using Notary. It's built using [Hugo][] and hosted on [Netlify][].
+---
 
-> **IMPORTANT:** For details on using Notary, such as quickstarts, tutorials, and other guides, see the [Notary documentation](https://notaryproject.dev/docs/). The rest of the README and other support files in this repo are for building and contributing to the website itself.
+## Introduction
+[**Checkout Notary Project's website!**](https://notaryproject.dev/)
 
-## Cloud build of the [notaryproject.dev][] website
+The Notary project enables multiple signatures of an [OCI Artifact](https://github.com/opencontainers/artifacts), such as container images, to be stored in an [OCI-compliant](https://github.com/opencontainers/oci-conformance) registry. It consists of various components, including requirements, scenarios, CLI tools, and libraries that implement these requirements.
```
Comment (iamsamirzon) [inline README.md:16]:
```suggestion
The Notary Project enables signatures for  [OCI Artifacts](https://github.com/opencontainers/artifacts), such as container images, to be stored in an [OCI-compliant](https://github.com/opencontainers/oci-conformance) registry. It consists of various components, including requirements, scenarios, CLI tools, and libraries that implement these requirements.
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,62 @@
...
 
-Visit [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
+This website serves as a comprehensive resource for information and documentation on Notary. Whether you're new to Notary or an experienced user, this is your go-to hub for all things related to Notary.
```
Comment (iamsamirzon) [inline README.md:18]:
```suggestion
This website serves as a comprehensive resource for information and documentation on the Notary Project. Whether you're new to the Notary Project or an experienced user, this is your go-to hub for all things.
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,62 @@
...
 
-## Contributing to the [notaryproject.dev][] website
-For more details, see [CONTRIBUTING](CONTRIBUTING.md)
+> For detailed information on how to use Notary, including quickstarts, tutorials, and other helpful guides, please refer to the comprehensive [Notary documentation](https://notaryproject.dev/docs/). The remaining content in this README file and other support files in this repository are specifically intended for building and contributing to the website.
```
Comment (iamsamirzon) [inline README.md:20]:
```suggestion
> For detailed information on how to use tools from The Notary Project ( such as Notation ), including quickstarts, tutorials, and other helpful guides, please refer to the comprehensive [Notary documentation](https://notaryproject.dev/docs/). The remaining content in this README file and other support files in this repository are specifically intended for building and contributing to the website.
```

File: README.md (hunk 1 above)
Comment (1Shubham7) [inline README.md:6]:
```suggestion
    <b>Welcome to the Notary Project! Our website serves as a comprehensive resource for information and documentation on Notary. Powered by Hugo and hosted on Netlify, it's your go-to hub for everything related to Notary.</b>
```

File: README.md (hunk 3 above)
Comment (1Shubham7) [inline README.md:18]:
```suggestion
This website serves as a comprehensive resource for information and documentation on Notary. Whether you're a newbie or an experienced Notary user, it is your ultimate go-to hub for everything about Notary.
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,62 @@
...
+To build the site and check links:
+```console
+npm run build
+```
+
+## Cloud-Based Website
+Visit this link to launch it - [Gitpod.io IDE-Notary](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev). Start building, previewing, and making changes to this repository.
+
+## Contributing
+We welcome contributors to contribute to the Notary website! We are still in the development phase, we eagerly encourage interested individuals to join us. Whether you're interested in updating code or improving documentation. If you encounter any issues or want to share any ideas, simply open an issue/PR or  let us know by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md).
```
Comment (1Shubham7) [inline README.md:53]:
```suggestion
We welcome contributors to contribute to the Notary website! We are still in the development phase, we wholeheartedly encourage interested individuals to join us. Whether you're interested in updating code or improving documentation. If you encounter any issues or want to share any ideas, simply open an issue/PR or  let us know by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md). Your valuable contributions will be greatly appreciated.
```

**Review by 1Shubham7 — COMMENTED** [review]
I have read the complete very thoroughly and it sounds almost perfect. These are not major changes but do add them if you feel they add some value.

File: README.md [hunk 6]
```diff
@@ -1,26 +1,62 @@
-# Notary Project Website
+<br />
+<p align="center">
+  <img src="https://github.com/sanjayk0508/notaryproject.dev/assets/102804548/63986f73-9669-4efe-a571-4d1935e0a3d4" alt="image 36">
+  <br />
+    <br />
+    <b>Welcome to the Notary Project! Our website, powered by Hugo and hosted on Netlify, serves as your go-to hub for all things related to Notary Project and its associated repositories.</b>
```
Comment (zr-msft) [inline README.md:6]:
```suggestion
    <b>Welcome to the Notary Project! This is the repository for the Notary Project website: https://notaryproject.dev/</b>
    <br/>
    <b>Our website, powered by Hugo and hosted on Netlify, and serves as your go-to hub for all things related to Notary Project and its associated repositories.</b>
```

File: README.md [hunk 7]
```diff
@@ -1,26 +1,62 @@
...
+    <br />
+    <br />
+</p>
 
-The [notaryproject.dev][] website, which contains information about the Notary Project as well as [documentation](https://notaryproject.dev/docs/) for using Notary. It's built using [Hugo][] and hosted on [Netlify][].
+---
 
-> **IMPORTANT:** For details on using Notary, such as quickstarts, tutorials, and other guides, see the [Notary documentation](https://notaryproject.dev/docs/). The rest of the README and other support files in this repo are for building and contributing to the website itself.
+## Introduction
+[**Checkout Notary Project's website!**](https://notaryproject.dev/)
```
Comment (zr-msft) [inline README.md:14]: Based on previous feedback, i don't think this is needed

File: README.md [hunk 8]
```diff
@@ -1,26 +1,62 @@
...
 
-## Cloud build of the [notaryproject.dev][] website
+The Notary Project enables signatures for  [OCI Artifacts](https://github.com/opencontainers/artifacts), such as container images, to be stored in an [OCI-compliant](https://github.com/opencontainers/oci-conformance) registry. It consists of various components, including requirements, scenarios, CLI tools, and libraries that implement these requirements.
 
-Visit [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
+This website serves as a comprehensive resource for information and documentation on the Notary Project. Whether you're new to the Notary Project or an experienced user, this is your go-to hub for all things.
```
Comment (zr-msft) [inline README.md:18]:
```suggestion
This website serves as a comprehensive resource for information and documentation on the Notary Project.
```
Last sentence was largely stated in the top section so you should remove it.

File: README.md [hunk 9]
```diff
@@ -1,26 +1,62 @@
...
 
-## Contributing to the [notaryproject.dev][] website
-For more details, see [CONTRIBUTING](CONTRIBUTING.md)
+> For detailed information on how to use tools from The Notary Project ( such as Notation ), including quickstarts, tutorials, and other helpful guides, please refer to the comprehensive [Notary documentation](https://notaryproject.dev/docs/). The remaining content in this README file and other support files in this repository are specifically intended for building and contributing to the website.
```
Comment (zr-msft) [inline README.md:20]:
```suggestion
> For detailed information on how to use tools from The Notary Project (such as [Notation](https://github.com/notaryproject/notation)), including quickstarts, tutorials, and other helpful guides, see the [Notary Project documentation](https://notaryproject.dev/docs/). The remaining content in this README file and other support files in this repository are specifically intended for building and contributing to the website.
```

File: README.md [hunk 10]
```diff
@@ -1,26 +1,62 @@
...
 
-## License
+## Installation
+1. **Fork** this repository and **clone** it to your development computer.
+2. From the terminal, navigate to the directory of your cloned repository.
+3. Add the main repository as a remote named **upstream**:
```
Comment (zr-msft) [inline README.md:25]:
```suggestion
3. Add the `notaryproject/notaryproject.dev` repository as a remote named **upstream**:
```

File: README.md [hunk 11]
```diff
@@ -1,26 +1,62 @@
...
+npm run serve
+```
 
-notaryproject.dev is licensed under an [Apache 2.0 license](./LICENSE).
+To build the site and check links:
+```console
+npm run build
+```
+
+## Cloud-Based Website
```
Comment (zr-msft) [inline README.md:49]:
```suggestion
## Cloud-Based IDE
```

File: README.md [hunk 12]
```diff
@@ -1,26 +1,62 @@
...
+Visit this link to launch it - [Gitpod.io IDE-Notary](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev). Start building, previewing, and making changes to this repository.
+
+## Contributing
+We welcome contributors to contribute to The Notary Project website! We are actively enhancing this site, so we encourage interested individuals to join us. Whether you're interested in updating code or improving documentation. If you encounter any issues or want to share any ideas, simply open an issue/PR or  let us know by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md).
```
Comment (zr-msft) [inline README.md:53]:
```suggestion
We welcome contributors to contribute to The Notary Project website! We are actively enhancing this site, so we encourage interested individuals to join us and contribute code and documentation. If you encounter any issues or want to share any ideas, please open an issue/PR. You can also provide feedback by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). For more information, see the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md).
```

File: README.md [hunk 13]
```diff
@@ -1,26 +1,62 @@
...
+
+## Join Us
+Join our growing community around the world! Check out our official [Blogs](https://notaryproject.dev/blog/). Join our [Slack community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/) - `#notaryproject` channel for any assistance and discussions. And follow us on [Twitter](https://twitter.com/NotaryProject).
```
Comment (zr-msft) [inline README.md:56]:
```suggestion
Join our growing community around the world! 
- Check out our official [Blogs](https://notaryproject.dev/blog/)
- Join our [Slack community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/) in the  `#notaryproject` channel
- Follow us on [Twitter](https://twitter.com/NotaryProject).
```

**Review by zr-msft — CHANGES_REQUESTED** [review]

File: README.md [hunk 14]
```diff
@@ -1,26 +1,67 @@
...
 
-Visit [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
+This website serves as a comprehensive resource for information and documentation on the Notary Project.
 
-## Contributing to the [notaryproject.dev][] website
-For more details, see [CONTRIBUTING](CONTRIBUTING.md)
+> For detailed information on how to use tools from The Notary Project (such as [Notation](https://github.com/notaryproject/notation)), including quickstarts, tutorials, and other helpful guides, see the [Notary Project documentation](https://notaryproject.dev/docs/). The remaining content in this README file and other support files in this repository are specifically intended for building and contributing to the website.
```
Comment (FeynmanZhou) [inline README.md:22]:
```suggestion
> You can learn about the overall Notary Project and organization governance from [Notary Project README](https://github.com/notaryproject/.github/blob/main/README.md). For detailed information on how to use tools from the Notary Project (such as [Notation](https://github.com/notaryproject/notation)), see the [Notary Project documentation](https://notaryproject.dev/docs/). The remaining content in this README file is specifically intended for building and contributing to the Notary Project website.
```

File: README.md [hunk 15]
```diff
@@ -1,26 +1,67 @@
...
 
-The [notaryproject.dev][] website, which contains information about the Notary Project as well as [documentation](https://notaryproject.dev/docs/) for using Notary. It's built using [Hugo][] and hosted on [Netlify][].
+---
 
-> **IMPORTANT:** For details on using Notary, such as quickstarts, tutorials, and other guides, see the [Notary documentation](https://notaryproject.dev/docs/). The rest of the README and other support files in this repo are for building and contributing to the website itself.
+## Introduction
+[**Checkout Notary Project's website!**](https://notaryproject.dev/)
 
-## Cloud build of the [notaryproject.dev][] website
+The Notary Project enables signatures for  [OCI Artifacts](https://github.com/opencontainers/artifacts), such as container images, to be stored in an [OCI-compliant](https://github.com/opencontainers/oci-conformance) registry. It consists of various components, including requirements, scenarios, CLI tools, and libraries that implement these requirements.
```
Comment (yizha1) [inline README.md:18]:
```suggestion
The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts. Notation Project specification and tooling provides signing and verification workflows for OCI artifacts, signature portability across OCI compliant registries, and integration with 3rd party key management solutions through a plugin model
```

**Review by yizha1 — CHANGES_REQUESTED** [review]
Thanks @sanjayk0508 , suggest updating the overview of Notary Project based on https://github.com/notaryproject/.github/pull/32

File: README.md [hunk 16]
```diff
@@ -1,26 +1,67 @@
...
 
-The [notaryproject.dev][] website, which contains information about the Notary Project as well as [documentation](https://notaryproject.dev/docs/) for using Notary. It's built using [Hugo][] and hosted on [Netlify][].
+---
 
-> **IMPORTANT:** For details on using Notary, such as quickstarts, tutorials, and other guides, see the [Notary documentation](https://notaryproject.dev/docs/). The rest of the README and other support files in this repo are for building and contributing to the website itself.
+## Introduction
+[**Checkout Notary Project's website!**](https://notaryproject.dev/)
 
-## Cloud build of the [notaryproject.dev][] website
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts. Notation Project specification and tooling provides signing and verification workflows for OCI artifacts, signature portability across OCI compliant registries, and integration with 3rd party key management solutions through a plugin model
```
Comment (iamsamirzon) [inline README.md:18]:
```suggestion
The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains by using authentic container images and other OCI artifacts. Notary Project specification and tooling provides signing and verification workflows for OCI artifacts, signature portability across OCI compliant registries, and integration with 3rd party key management solutions through a plugin model
```

**Review by iamsamirzon — DISMISSED** [review]

File: README.md [hunk 17]
```diff
@@ -1,26 +1,70 @@
...
+To build the site and check links:
+```console
+npm run build
+```
+
+## Cloud-Based IDE
+Visit this link to launch it - [Gitpod.io IDE-Notary](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev). Start building, previewing, and making changes to this repository.
+
+## Contributing
+We welcome contributors to contribute to The Notary Project website! We are actively enhancing this site, so we encourage interested individuals to join us and contribute code and documentation. If you encounter any issues or want to share any ideas, please open an issue/PR. You can also provide feedback by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). For more information, see the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md).
```
Comment (zr-msft) [inline README.md:58]:
```suggestion
We welcome contributors to contribute to the Notary Project website! We are actively enhancing this site, so we encourage interested individuals to join us and contribute code and documentation. If you encounter any issues or want to share any ideas, please open an issue/PR. You can also provide feedback by joining our [Slack Community](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/). For more information, see the [contribution guide](https://github.com/notaryproject/notaryproject.dev/blob/main/CONTRIBUTING.md).
```

**Review by zr-msft — CHANGES_REQUESTED** [review]

**Review by zr-msft — APPROVED** [review]

**Comment by zr-msft** [comment]
@yizha1  @FeynmanZhou can you review this PR please and I'll handle merging

**Review by yizha1 — APPROVED** [review]
Thanks @sanjayk0508  LGTM

**Pull request merged**